gridstatus/caiso/caiso.py

```diff
@@ -2435,3 +2443,372 @@ def get_interval_nomogram_branch_shadow_prices_real_time_5_min(
+                    "OperatorInstruction System MW": "curt_daily_oi_system_mw",
+                },
+            ),
+            # Fuel-specific hourly curtailment data
+            (
+                "solar_curtailment_maximum_hourly",
+                hourly_timestamps,
+                1,
+                "hour",
+                {
+                    "Economic Local MW": "curt_hr_max_solar_econ_local_mw",
+                    "Economic System MW": "curt_hr_max_solar_econ_system_mw",
+                    "SelfSchCut Local MW": "curt_hr_max_solar_ss_local_mw",
+                    "SelfSchCut System MW": "curt_hr_max_solar_ss_system_mw",
+                    "OperatorInstruction Local MW": "curt_hr_max_solar_oi_local_mw",
+                    "OperatorInstruction System MW": "curt_hr_max_solar_oi_system_mw",
+                },
+            ),
+            (
+                "wind_curtailment_maximum_hourly",
+                hourly_timestamps,
+                1,
+                "hour",
+                {
+                    "Economic Local MW": "curt_hr_max_wind_econ_local_mw",
+                    "Economic System MW": "curt_hr_max_wind_econ_system_mw",
+                    "SelfSchCut Local MW": "curt_hr_max_wind_ss_local_mw",
+                    "SelfSchCut System MW": "curt_hr_max_wind_ss_system_mw",
+                    "OperatorInstruction Local MW": "curt_hr_max_wind_oi_local_mw",
+                    "OperatorInstruction System MW": "curt_hr_max_wind_oi_system_mw",
+                },
+            ),
+            (
+                "solar_curtailment_total_hourly",
+                hourly_timestamps,
+                1,
+                "hour",
+                {
+                    "Economic Local MWH": "curt_hr_tot_solar_econ_local_mwh",
+                    "Economic System MWH": "curt_hr_tot_solar_econ_system_mwh",
+                    "SelfSchCut Local MWH": "curt_hr_tot_solar_ss_local_mwh",
+                    "SelfSchCut System MWH": "curt_hr_tot_solar_ss_system_mwh",
+                    "OperatorInstruction Local MWH": "curt_hr_tot_solar_oi_local_mwh",
+                    "OperatorInstruction System MWH": "curt_hr_tot_solar_oi_system_mwh",
+                },
+            ),
+            (
+                "wind_curtailment_total_hourly",
+                hourly_timestamps,
+                1,
+                "hour",
+                {
+                    "Economic Local MWH": "curt_hr_tot_wind_econ_local_mwh",
+                    "Economic System MWH": "curt_hr_tot_wind_econ_system_mwh",
+                    "SelfSchCut Local MWH": "curt_hr_tot_wind_ss_local_mwh",
+                    "SelfSchCut System MWH": "curt_hr_tot_wind_ss_system_mwh",
+                    "OperatorInstruction Local MWH": "curt_hr_tot_wind_oi_local_mwh",
+                    "OperatorInstruction System MWH": "curt_hr_tot_wind_oi_system_mwh",
+                },
+            ),
```

Comment:
These are the values used in the new curtailment method. I went ahead and parsed everything in the renewables report if we want to use it for other datasets.